### ISSUE TITLE
Reversible Possible?

### ISSUE THREAD

**Issue by trinadhkoya** [Arjun-sna/android-swipecards-view#17]
how to make those cards reversable.I want the child views to be shown above the (parent/foreground) card!!

**Comment by vinodsvist252** [comment]
Yes trinadhyoka, reversable was possible.While doing onCardExitLef store that particular pojo object in a seperate arraylist like  UserlistPojo pojo = (UserlistPojo) dataObject;
   UserList_changed.add(pojo);   ........and at the time of calling the rewind button check like this

                 if (UserList_changed.size() > 0) {
                            mSwipeView.START_STACK_FROM = 0;
                            mSwipeView.MAX_VISIBLE = 3;
                            mSwipeView.LAST_OBJECT_IN_STACK = 0;
                            mSwipeView.MIN_ADAPTER_STACK = 6;

                            rewind_status = "rewind";

                            for (int i = 0; i < UserList_changed.size(); i++) {
                                UserList.remove(UserList_changed.get(i));
                                System.out.println("----name at position---" + i + " " + 
                                   UserList_changed.get(i).getName());
                            }

                            UserList.add(0, UserList_changed.get(UserList_changed.size() - 1));
                            UserList.get(0).setRewind_status(rewind_status);
                            array_value = array_value + 1;
                            System.out.println("----name at last postion--" + 
                            UserList_changed.get(UserList_changed.size() - 1).getName());

                            UserList_changed.remove(UserList_changed.get(UserList_changed.size() - 1));
                            adapter.notifyDataSetChanged();

..............there i was storing the setting the rewind status for every object and based on that in adapter i was calling and adding animations for that particular card and after adding i am setting the rewind status for that particular object.
